framework/core/tests/integration/api/discussions/CreateTest.php

```diff
@@ -138,7 +138,40 @@ public function can_create_discussion()
      */
     public function can_create_discussion_with_current_lang_slug_transliteration()
     {
-        $this->app()->getContainer()->make('flarum.locales')->setLocale('zh');
+        // Forum default is traditional Chinese.
+        $this->app()->getContainer()->make('flarum.settings')->set('default_locale', 'zh');
```

Comment:
let's use the existing method to change settings in tests:
```suggestion
        $this->setting('default_locale', 'zh');
```

Reply:
Oh wow I copied that from another test 🙈